server/helpers/valueAddedServices.js

```diff
@@ -0,0 +1,26 @@
+import { VALUE_ADDED_SERVICE_ACCESS_LEVEL } from './constants';
+
+const VALUE_ADDED_SERVICES = {
+  SURVEY_PLAN_INVESTIGATION: {
+    name: 'Survey Plan Investigation',
+    price: 30_000,
+    type: VALUE_ADDED_SERVICE_ACCESS_LEVEL.USER,
```

Comment:
As discussed, this is more related to Property than User. Please fix for others.
```suggestion
    type: VALUE_ADDED_SERVICE_ACCESS_LEVEL.PROPERTY,
```